rules_gapic/cpp/cc_gapic.bzl

```diff
@@ -0,0 +1,114 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@com_google_api_codegen//rules_gapic:gapic.bzl", "gapic_srcjar", "unzipped_srcjar")
+
+def _cc_gapic_postprocessed_srcjar_impl(ctx):
```

Comment:
It's not really a _jar_ since it's not Java. My immediate thought was 'tarball', but it's not a tarball either. Zipball? Archive? Archive seems helpful and descriptive. Or is the `.srcjar` output file name required by the tool?

Reply:
It is not `jar`, it is `srcjar` and it is a convention, and is used by other rules. java_library even accepts .srcjar (not zip or jar) as input for `src` (to support undefined on analysis phase set of input files).